pages.id/windows/chkdsk.md

```diff
@@ -0,0 +1,20 @@
+# chkdsk
+
+> Memeriksa dan mencari kesalahan dalam sebuah sistem file dan metadata volume penyimpanan.
+> Informasi lebih lanjut: <https://docs.microsoft.com/windows-server/administration/windows-commands/chkdsk>.
+
+- Memeriksa sebuah volume penyimpanan berdasarkan huruf drive (diakhiri dengan titik dua), lokasi pemasangan, atau nama volume:
+
+`chkdsk {{volume}}`
```

Comment:
Why `volume` doesn't need a translation?

Reply:
Actually, there is `ruang`, the synonym of `volume` in Indonesian, However, `ruang penyimpanan` can also be interpreted as `storage room` aka. `warehouse`, so it might be confusing for some people.

After checking across some official documentation from Microsoft and BlankOn (an Indonesian-based Linux distro) `ruang penyimpanan` is preferred since `volume penyimpanan` can also be interpreted as the size of the storage volume, so I will change them to `ruang penyimpanan` instead.

Reply:
Fixed on https://github.com/tldr-pages/tldr/pull/6067/commits/7c5f4d6eacc12c48c4e8905854dcb59c4e39eb13.

Reply:
Thanks @reinhart1010